### PR TITLE
Release 0.15.0 fixes

### PR DESCRIPTION
**Pull request by G1gg1L3s**
This PR contains update docs for the new Themis release.

Creating this PR now because I have a felling that we could forget something. So everytime you remember that we have to update something, you can go and write it here and somebody (probably me) will dot it.

### CODE REVIEW

**Comment by vixentael** [comment]
this one too
https://github.com/cossacklabs/product-docs/pull/265

File: content/themis/installation/installation-from-sources.md [hunk 1]
```diff
@@ -37,15 +37,6 @@ as it contains the necessary header files.
 
 In either case, we **strongly** recommend that you use the most recent version of these packages.
 
-{{< hint warning >}}
-**Note:**
-At the moment Themis is not fully compatible with OpenSSL 3.0.
```
Comment (vixentael) [inline content/themis/installation/installation-from-sources.md:42]: we should not just remove it, we should write smth like "Starting from Themis 0.15, it supports OpenSSL 3.0 as well as OpenSSL 1.1.1.." blahblahblah

Comment (G1gg1L3s) [reply content/themis/installation/installation-from-sources.md:42]: oh, yeah, good point

File: content/themis/installation/installation-from-sources.md [hunk 2]
```diff
@@ -202,10 +193,10 @@ OpenSSL is the default cryptographic engine on most platforms.
 Themis will automatically detect and use OpenSSL installed in your system.
 If you wish to explicitly require OpenSSL, set `ENGINE=openssl` when building Themis.
 
-{{< hint warning >}}
+{{< hint info >}}
 **Note:**
-At the moment Themis supports only OpenSSL 1.0.2 and 1.1.x branches.
-OpenSSL 3.0 is still not fully supported.
+
+Starting from version [0.15.0](https://github.com/cossacklabs/themis/releases/tag/0.15.0), Themis supports OpenSSL 3.0. Consider using it instead of OpenSSL 1.1.1, as its end of life is [September 2023](https://www.openssl.org/blog/blog/2023/03/28/1.1.1-EOL/).
```
Comment (vixentael) [inline content/themis/installation/installation-from-sources.md:199]: nice!

Comment (Lagovas) [inline content/themis/installation/installation-from-sources.md:199]: let's mention that we tested with 3.0.2 and 3.0.8 if I'm not mistaken. Or can I ask @G1gg1L3s to look on openssl version in the packages of supported OSs in buildbot? which versions are installed to supported debian/ubuntu/centos that we test on buildbot?

Comment (G1gg1L3s) [reply content/themis/installation/installation-from-sources.md:199]: For now, only ubuntu:jammy has the openssl 3.0.2. Once we setup debian:bookworm, we would get openssl 3.0.9. So, yeah, let's put the tag on.

**Review by vixentael — APPROVED** [review]

**Pull request merged**